infer/lib/python/inferlib/capture/gradle.py

```diff
@@ -31,9 +30,136 @@ def gen_instance(*args):
 create_argparser = util.base_argparser(MODULE_DESCRIPTION, MODULE_NAME)
 
 
+def extract_filepath(parts):
+    pos = 0
```

Comment:
Great, thanks for the context. I agree that we shouldn't worry about use-cases that are purely hypothetical for now.